src/main/java/duke/exception/FileNotFoundException.java

```diff
@@ -0,0 +1,4 @@
+package duke.exception;
```

Comment:
Is this file necessary? Would just like to check!

Reply:
Yep! This exception is used in `Storage.java`. I think I would probably add more explanatory comments on the exception classes I've created, so other devs know what these are for.